### PR TITLE
Showing media size, height and width in medialibrary

### PR DESCRIPTION
**Pull request by hbaptiste**
Showing media size, height and width in medialibrary

### CODE REVIEW

File: src/tb/component/medialibrary/main.js [hunk 1]
```diff
@@ -155,11 +155,11 @@ define(
 
                 addButtons: function () {
                     var self = this;
-                    this.dialog.addButton("Add & Close", function () {
+                    this.dialog.addButton(trans("add_and_close"), function () {
                         self.triggerEvent = true;
                         self.close();
                     });
-                    this.dialog.addButton("Cancel", function () {
+                    this.dialog.addButton(trans("cancel"), function () {
```
Comment (mickaelandrieu) [inline src/tb/component/medialibrary/main.js]: thanks @hbaptiste :+1:

File: src/tb/i18n/en_US/global.json [hunk 2]
```diff
@@ -74,7 +75,7 @@
     "general_settings": "General Settings",
     "group": "Group",
     "groups": "Groups",
-
+    "height": "H",
```
Comment (mickaelandrieu) [inline src/tb/i18n/en_US/global.json]: ? ``height : height`` and ``"H" : "H"`` imo

File: src/tb/i18n/en_US/global.json [hunk 3]
```diff
@@ -168,7 +170,7 @@
     "users_management": "Users management",
 
     "validate": "Validate",
-
+    "width": "W",
```
Comment (mickaelandrieu) [inline src/tb/i18n/en_US/global.json]: same here, ask to @ndufreche for conventions in translation keys (@nndufreche: can you write a little docs about it ?)

File: src/tb/i18n/en_US/global.json (hunk 2 above)
Comment (ndufreche) [inline src/tb/i18n/en_US/global.json]: 'h': 'H'

File: src/tb/i18n/en_US/global.json (hunk 3 above)
Comment (ndufreche) [inline src/tb/i18n/en_US/global.json]: 'w': 'W'

**Pull request merged**